src/test/scala/nl.knaw.dans.easy.bagstore/server/BagsServletSpec.scala

```diff
@@ -206,4 +208,58 @@ class BagsServletSpec extends TestSupportFixture
       body shouldBe s"Item $itemId not found"
     }
   }
+
+  it should "fail when done on an inactive/ hidden bag" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID") {
+      status shouldBe 410
+      body.lines.mkString should include(s"Tried to retrieve an inactive bag: $bagID")
+    }
+  }
+
+  it should "fail, returning a 410, when done on an item within an inactive/ hidden bag" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID/bag-info.txt") {
+      status shouldBe 410
+      body.lines.mkString should include(s"Tried to retrieve an inactive bag: $bagID")
+    }
+  }
+
+  it should "return a 404 non-existing item within an inactive/ hidden bag is requested" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID/bag-info6.txt") {
+      status shouldBe 404
+      body.lines.mkString should include(s"Item $bagID/bag-info6.txt not found")
+    }
+  }
+
+  // this calls enumFiles
+  it should " fail when done on an inactive/ hidden bag when headers text/plain is provided" in {
+    val bagID = "01000000-0000-0000-0000-000000000001"
+    bagStore1.deactivate(BagId(UUID.fromString(bagID))) shouldBe a[Success[_]]
+    get(s"/$bagID", headers = Map("Accept" -> "text/plain")) {
+      status shouldBe 410
```

Comment:
here and in 2 other places in this spec, you don't have tests for `body`, even though I suspect they're not empty. Please add them.